qemu/target-mips/cpu.c

```diff
@@ -151,15 +151,17 @@ void mips_cpu_register_types(void *opaque)
         
         sizeof(MIPSCPUClass),
         sizeof(MIPSCPU),
-		opaque,
+        opaque,
         
-		mips_cpu_initfn,
-		NULL,
-		NULL,
+        mips_cpu_initfn,
+        NULL,
+        NULL,
+
+        NULL,
```

Comment:
Yes it was this missing NULL.
This is one of those const initialisations though.
You could remove the const, but then you are trading readability for performance.